providers/apache/livy/pyproject.toml

```diff
@@ -68,6 +69,7 @@ dev = [
     "apache-airflow",
     "apache-airflow-task-sdk",
     "apache-airflow-devel-common",
+    "apache-airflow-providers-common-compat",
```

Comment:
Nit/not blocking, but I don't like this:

It feels that we are putting OL specific code in the common-compat module. It should be reserved for compatibilty across versions. I really don't like provider specific code living in here.

Reply:
I removed the OL-specific code from that module. Updated it simply because it was there, but I agree there's no reason for them to be there. 

On the other hand, OL-specific _imports_ are one of the reasons for common.compat - instead of having to try/except everywhere in providers when trying to import something from OL, you can import from common.compat and have no-op version if it's not present.